mlflow/tracking/_tracking_service/utils.py

```diff
@@ -171,10 +171,39 @@ def _get_databricks_rest_store(store_uri, **_):
     return RestStore(partial(get_databricks_host_creds, store_uri))
 
 
+def _get_databricks_uc_rest_store(store_uri, **_):
+    from mlflow.exceptions import MlflowException
+    from mlflow.version import VERSION
+
+    global _tracking_store_registry
+    supported_schemes = list(
+        filter(
+            lambda scheme: scheme != _DATABRICKS_UNITY_CATALOG_SCHEME,
+            list(_tracking_store_registry._registry.keys()),
+        )
+    )
```

Comment:
I think list comprehension is simpler:

```suggestion
    supported_schemes = [
        scheme
        for scheme in _tracking_store_registry._registry.keys()
        if scheme != _DATABRICKS_UNITY_CATALOG_SCHEME
    ]
```